File: 140519/test-student.c
#include <stdio.h>

typedef struct {
    char *name;
    int grade;
}STUDENT;

int main(int argc, char *argv[]) {
    STUDENT s1, s2;
    s1.name = "suzuki";
    s2.name = "yamada";

    s1.grade = 3;
    s2.grade = 4;

    printf("student name = %s (%d)\n", s1.name, s1.grade);
    printf("student name = %s (%d)\n", s2.name, s2.grade);

    return 0;
}

File: 140519/test-student.py
#-*- coding:utf-8 -*-

class Student:
    def __init__(self, name):
        self.name = name
    def setGrade(self, grade):
        self.grade = grade


def main():
    s1 = Student("yamada")
    s2 = Student("suzuki")

    s1.setGrade(1)
    s2.setGrade(2)

    format = "student name = {0} ({1})"
    print format.format(s1.name, s1.grade)
    print format.format(s2.name, s2.grade)

main()
